packages/shared/__tests__/button.test.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
```

Comment:
Try to put your tests in the same way as your architecture, for example (__tests__/src/components/ButtonLogin.test.jsx)
This is for a better organazed project.